### ISSUE TITLE
'margipose.data.mpi_inf_3dhp', 'sequence_info.json'

### ISSUE THREAD

**Issue by sunmengnan** [anibali/margipose#31]
could you upload the files above extracted from "mpii_get_sequence_info.m", i cannot get these files due to matlab limatations, thx.

**Comment by anibali** [comment]
That file is a part of the repository, you can find it here: https://github.com/anibali/margipose/blob/master/src/margipose/data/mpi_inf_3dhp/sequence_info.json

**Comment by sunmengnan** [comment]
in Constants = {
    # Training set sequences: (subject_id, sequence_id)
    'train_seqs': [
        (1, 1), (1, 2), (2, 1), (2, 2), (3, 1), (3, 2), (4, 2),
        (5, 1), (5, 2), (6, 1), (6, 2), (7, 1), (7, 2), (8, 1),
    ],
    # Validation set sequences: (subject_id, sequence_id)
    'val_seqs': [
        (4, 1), (8, 2)
    ],
    # Camera IDs used for training/validation sets (same as VNect)
    'vnect_cameras': [0, 1, 2, 4, 5, 6, 7, 8],
    # Total number of cameras
    'n_cameras': 14,
    # Per-sequence information extracted from "mpii_get_sequence_info.m"
    'seq_info': json.loads(read_text('margipose.data.mpi_inf_3dhp', 'sequence_info.json')),
    # Root joint index (pelvis) for training/validation data
    'root_joint': 4,
    # Videos with known problems
    'blacklist': {
        'S6/Seq2': [2],  # imageSequence/video_2.avi is too short
    },
}
only S6/Seq2 in the blacklist, is it only for test?

**Comment by anibali** [comment]
S6/Seq2 is a training sequence, but there's something wrong with the `video_2.avi` file so that particular view is skipped for that sequence when preprocessing the dataset.
